compiler/src/dotty/tools/io/ZipArchive.scala

```diff
@@ -199,7 +199,7 @@ final class FileZipArchive(jpath: JPath) extends ZipArchive(jpath) {
 final class ManifestResources(val url: URL) extends ZipArchive(null) {
   def iterator(): Iterator[AbstractFile] = {
     val root     = new DirEntry("/", null)
-    val dirs     = new mutable.HashMap[String, DirEntry]; dirs.put("/", root)
+    val dirs     = mutable.HashMap[String, DirEntry]("/" -> root)
```

Comment:
Was this change intended in this PR?

Reply:
This is not intended, I'll move it to #9644.